WordPress/Classes/Models/ReaderPostContentProvider.h

```diff
@@ -8,11 +8,12 @@ typedef NS_ENUM(NSUInteger, SourceAttributionStyle) {
 };
 
 @protocol ReaderPostContentProvider <PostContentProvider>
-- (NSURL *)siteIconForDisplayOfSize:(NSInteger)size;
 - (SourceAttributionStyle)sourceAttributionStyle;
 - (NSString *)sourceAuthorNameForDisplay;
 - (NSURL *)sourceAuthorURLForDisplay;
 - (NSURL *)sourceAvatarURLForDisplay;
+- (NSString *)siteIconURL;
+- (NSString *)blogURL;
```

Comment:
Maybe we can also take the opportunity to add some nullability annotations since we are modifying obj-c code.